tdrs-backend/tdpservice/settings/common.py

```diff
@@ -110,7 +110,7 @@ class Common(Configuration):
     USE_I18N = False
     USE_L10N = True
     USE_TZ = True
-    LOGIN_URL = "/v1/login/oidc"
+    LOGIN_URL = FRONTEND_BASE_URL
```

Comment:
Updated this to the `FRONTEND_BASE_URL` per @raftmsohani to account for #1520

Reply:
confirming that `https://tdp-backend-qasp.app.cloud.gov/admin/` redirects me to frontend login, as we'd hoped.